compiler/il/OMRSymbolReference.hpp

```diff
@@ -47,6 +47,11 @@ namespace OMR { typedef OMR::SymbolReference SymbolReferenceConnector; }
 #include "infra/BitVector.hpp"
 #include "infra/Flags.hpp"
 
+typedef enum {
+   useDefAliasSet,
+   UseOnlyAliasSet
```

Comment:
can you please make the capitalization consistent?

Reply:
Fixed. The enum in the documentation still has this problem but I will fix those in https://github.com/eclipse/omr/pull/4717.